functional_tests/scylla_operator/test_functional.py

```diff
@@ -194,9 +212,9 @@ def get_pod_names_and_ips(cluster_name: str, namespace: str):
             for current_namespace in pods_data.keys():
                 for pod_name, pod_ip in pods_data[current_namespace].items():
                     cqlsh_cmd = "SELECT JSON peer, data_center, rack, rpc_address FROM system.peers;"
-                    cqlsh_results = kubectl(
-                        f"exec {pod_name} -- /bin/cqlsh -e \"{cqlsh_cmd}\"",
-                        namespace=current_namespace).stdout.split("---\n")[-1].split("\n")
+                    cqlsh_results = (
+                        kubectl(f'exec {pod_name} -- /bin/cqlsh -e "{cqlsh_cmd}"', namespace=current_namespace).stdout.split("---\n")[-1].split("\n")
+                    )
```

Comment:
It created too long line.
One more case of counter-productivity.
Probably there should be some config option for setting the limit for line length it tries to apply.